app/controllers/concerns/hydra/collections_controller_behavior.rb

```diff
@@ -175,7 +175,9 @@ def move_members_between_collections
     # include filters into the query to only include the collection memebers
     def include_collection_ids(solr_parameters, user_parameters)
       solr_parameters[:fq] ||= []
-      solr_parameters[:fq] << Solrizer.solr_name(:collection, :facetable)+':"'+@collection.id+'"'
+
+      # http://search-lucene.com/javadoc/lucene/core/org/apache/lucene/search/DocValuesTermsFilter.html
+      solr_parameters[:fq] << "{!terms f=id method=docValuesTermsFilter}#{@collection.member_ids.join(',')}"
     end
```

Comment:
@jcoyne Have you tried this change with large quantities of ids?  We ran into serious issues with ArchiveSphere when we had thousands of files in a collection.  We hit string limits, and http limits.  I'm a bit concerned about going backward.

I understand the reason you are doing this, but I'm not sure going back to what did not work in the past is the right decision.